pkg/sbom/io/encode_test.go

```diff
@@ -562,6 +654,9 @@ func TestEncoder_Encode(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			uuid.SetFakeUUID(t, "3ff14136-e09f-4df9-80ea-%012d")
 
+			if tt.addBOMFunc != nil {
+				tt.report.BOM = tt.addBOMFunc()
```

Comment:
Why don't we add BOM [here](https://github.com/aquasecurity/trivy/pull/6808/files#diff-c5b3c454bb8e04604ef01e02f61e67f9f58631781e30671d252acb293a8d6897R542)?

Reply:
In this case we will create `BOM` before running test case and `fakeUUID` not be used.

But i think we can just create another `SetFakeUUID` with different format to create the BOM.
Check https://github.com/aquasecurity/trivy/pull/6808/commits/068b89048a50e53075546e18fef94104d5c710aa please